### PR TITLE
[15.0][MIG]account_fiscal_year: Migration from 14.0 to 15.0

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**

### CODE REVIEW

File: account_fiscal_year/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Account Fiscal Year",
     "summary": "Create Account Fiscal Year",
-    "version": "14.0.1.1.1",
+    "version": "15.0.1.1.1",
```
Comment (JoanMForgeFlow) [inline account_fiscal_year/__manifest__.py]: Please change to 15.0.1.0.0

**Review by JoanMForgeFlow — CHANGES_REQUESTED** [review]

File: account_fiscal_year/__manifest__.py (hunk 1 above)
Comment (AlvaroTForgeFlow) [reply account_fiscal_year/__manifest__.py]: Done.

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM!

**Review by Jortolsa-S73 — APPROVED** [review]
LGTM

**Comment by BT-anieto** [comment]
Could you please check ci so that it can be merged?

**Comment by AaronHForgeFlow** [comment]
It is already merged. Closing.

**Pull request closed without merging**